controls/cis_al2023.yml

```diff
@@ -0,0 +1,2176 @@
+
+  - id: 4.6.2
+    title: Ensure system accounts are secured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - no_password_auth_for_systemaccounts
+      - no_shelllogin_for_systemaccounts
+
+  - id: 4.6.3
+    title: Ensure default user shell timeout is 900 seconds or less (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_tmout
+      - var_accounts_tmout=15_min
+
+  - id: 4.6.4
+    title: Ensure default group for the root account is GID 0 (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_root_gid_zero
+
+  - id: 4.6.5
+    title: Ensure default user umask is 027 or more restrictive (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - accounts_umask_etc_bashrc
+      - accounts_umask_etc_login_defs
+      - accounts_umask_etc_profile
+      - var_accounts_user_umask=027
+
+  - id: 4.6.6
+    title: Ensure root password is set (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - ensure_root_password_configured
+      - no_empty_passwords_etc_shadow
+
+  - id: 5.1.1.1
+    title: Ensure rsyslog is installed (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - package_rsyslog_installed
+
+  - id: 5.1.1.2
+    title: Ensure rsyslog service is enabled (Manual)
+    levels:
+      - l1_server
+    status: manual
+    rules:
+      - service_rsyslog_enabled
+
+  - id: 5.1.1.3
+    title: Ensure journald is configured to send logs to rsyslog (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - journald_forward_to_syslog
+
+  - id: 5.1.1.4
+    title: Ensure rsyslog default file permissions are configured (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - rsyslog_filecreatemode
+
+  - id: 5.1.1.5
+    title: Ensure logging is configured (Manual)
+    levels:
+      - l1_server
+    status: manual
+
+  - id: 5.1.1.6
+    title: Ensure rsyslog is configured to send logs to a remote log host (Manual)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - rsyslog_remote_loghost
+
+  - id: 5.1.1.7
+    title: Ensure rsyslog is not configured to recieve logs from a remote client (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - rsyslog_nolisten
+
+  - id: 5.1.2.1.1
+    title: Ensure systemd-journal-remote is installed (Manual)
+    levels:
+      - l1_server
+    status: manual
+
+  - id: 5.1.2.1.2
+    title: Ensure systemd-journal-remote is configured (Manual)
+    levels:
+      - l1_server
+    status: manual
+
+  - id: 5.1.2.1.3
+    title: Ensure systemd-journal-remote is enabled (Manual)
+    levels:
+      - l1_server
+    status: manual
+
+  - id: 5.1.2.1.4
+    title: Ensure journald is not configured to recieve logs from a remote client (Automated)
```

Comment:
Replace with "title: Ensure journald is not configured to receive logs from a remote client (Automated)".

Reply:
cant spot the diff here

Reply:
recieve → receive

Reply:
thanks